src/main/webapp/includes/download-box-openfire.jspf

```diff
@@ -46,8 +46,8 @@
         <div class="ignite_downloads_content">
             <span class="ignite_downloads_info">
                 <a href="<%=request.getContextPath()%>/projects/openfire/plugins.jsp"><strong>Plugins</strong></a> |
-                <a href="<%=request.getContextPath()%>/builds/openfire/docs/latest/README.html">Readme</a> |
-                <a href="<%=request.getContextPath()%>/builds/openfire/docs/latest/LICENSE.html">License</a> |
+                <a href="https://github.com/igniterealtime/Openfire/releases"><strong>Old versions</strong></a>
```

Comment:
I think you are missing a "|" separator here.

Reply:
Thanks. Added